etna/lib/etna/application.rb

```diff
@@ -59,6 +60,21 @@ def configure(opts)
     end
   end
 
+  def setup_yabeda
+    Yabeda.configure!
```

Comment:
Etna applications can overload this method and call out to functions that may configure yabeda metrics, calling super to also trigger the crucial `Yabeda.configure!` step.  Yabeda itself is just a ruby library (https://github.com/yabeda-rb/yabeda
) that provides the DSL for setting up prometheus or statsd metrics in a coherent way.  Examples below.